scripts/start_docker_marqo.sh

```diff
@@ -15,7 +15,7 @@ docker rm -f marqo;
 set -x
 docker run -d --name marqo -it -p 8882:8882 \
     -e MARQO_ENABLE_BATCH_APIS=TRUE \
-    -e "MARQO_MAX_CPU_MODEL_MEMORY=1.6" \
+    -e "MARQO_MAX_CPU_MODEL_MEMORY=15" \
```

Comment:
Same, let move your full model test to cuda test only. CPU are for lightweight models